src/lib/index.ts

```diff
@@ -1,6 +1,5 @@
 export type {
 	IConfig,
-	IContext,
 	IMutableContext,
```

Comment:
I believe we should keep exporting this. E.g. [React SDK](https://github.com/Unleash/unleash-react-sdk/blob/main/src/index.ts#L5)
```suggestion
	IContext,
	IMutableContext,
```